### DISCUSSION TITLE
Industry Model with wastedisagg specs, useeior v1.0.0 ( "USEEIOv2.0.1-411")

### DISCUSSION THREAD

**Discussion by Georgina5764** [USEPA/useeior#284] [Q&A] [answered]
Hi,
When trying to obtain the Industry model equivalent of the excel file release, with the waste disaggregation specs, in [DATA CATALOG - USEEIO v2.0.1-411](https://catalog.data.gov/dataset/useeio-v2-0-1-411#:~:text=0.1%2D411,-Metadata%20Updated%3A%20January&text=This%20dataset%20provides%20the%20waste,metadata%20for%20the%20USEEIO%20v2.) : [USEEIOv2.0.1-411.xlsx](https://catalog.data.gov/dataset/useeio-v2-0-1-411/resource/09ec4bd2-674d-416b-bb1b-0ab9b5e8c3e9) I run into the error "Error in calculateBalancedDomesticTables(model, disagg, balancedDisaggFullUse) : 
  object 'balancedDisaggFullUse' not found". Would you have excel files for the Industry model with waste disaggregation and this specific version which could be accessed ?
Separately, I believe this was the latest release with an excel output provided, is there any plan to do a similar release for the latest versions ?
Thanks in advance

**Comment by bl-young** [comment]
Hi can you clarify the code you are running that generates that error? In our model testing, generating a detail industry model with the waste disaggregation proceeds without error.

Yes, at this time the 2.0.1-411 model remains the most recently published full model.

**Reply by Georgina5764** [reply]
Thanks, I ran  "USEEIOv2.0.1-411.yml" only changing CommodityorIndustryType: "Industry" in version 1.0.0 [v1.0.0](https://github.com/USEPA/useeior/tree/v1.0.0) as specified in [DATA CATALOG - USEEIO v2.0.1-411](https://catalog.data.gov/dataset/useeio-v2-0-1-411#:~:text=0.1%2D411,-Metadata%20Updated%3A%20January&text=This%20dataset%20provides%20the%20waste,metadata%20for%20the%20USEEIO%20v2.). If you have an output for Industry with waste disaggregation, would it maybe be possible to make it available as an excel file ?

**Reply by bl-young** [reply, marked as answer]
Oh I see, I think in [v1.2.1](https://github.com/USEPA/useeior/releases/tag/v1.2.1) we resolved an error related to industry models with disaggregation. I would recommend using the latest version of useeior to rebuild the model, currently v1.4.0. Note that BEA does make updates to their data on occasion, so we try to maintain the latest versions, and therefore a model generated with later versions of the package may have slightly different results.

**Reply by Georgina5764** [reply]
I see, alright thanks appreciate the fast reply ! In case it is feasible it would be great to get excel files outputs for the next releases as they are easier to source/trace back

**Reply by bl-young** [reply]
We have an increasingly long list of models and model types so we often can't re-release or release different versions of our models in all formats. However we have been and continue to improve our testing and validation procedures to ensure that published models can be adapted in various ways and still be built correctly. However, we encourage users to continue to take advantage of these discussion boards and other avenues to reach out with questions or if unexpected issues arise!
